app/style/MiscComponents.less

```diff
@@ -77,9 +78,10 @@
 
 .key-blue-button:active {
   box-shadow: 0 0 0 0 rgba(0, 0, 0, 0.2);
+  background-color: #0025cc;
 }
 
-.key-blue-button-disabled {
+.key-blue-button-disabled, .key-blue-button[disabled] {
```

Comment:
Not sure where the issue is arising from but the buttons' text looks to be larger or possibly bolded compared to existing

Reply:
I did that because all the buttons in decrediton are not a real buttons like `<button></button>` or `<input type="submit" />`, they are a simple `div` with an `onClick` event.

Because for my form I'm using the `onSubmit` event, with a reduxform, I have to use a real button, and then I can apply the disable style with a `disabled` key inside the input. Easier than toggling the className I think

Reply:
I think we should switch to buttons as much as possible.

I am a favor of semantic markup, at least at a basic level.